### PR TITLE
Added VB example for NegotiateStream

### PR DESCRIPTION
**Pull request by obelink**
This PR contributes to:
https://github.com/dotnet/dotnet-api-docs/issues/2069

Added the VB example for NegotiateStream.

### CODE REVIEW

File: snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb [hunk 1]
```diff
@@ -0,0 +1,112 @@
+Imports System.Text
+Imports System.Net.Sockets
+Imports System.Net.Security
+Imports System.Net
+'<snippet0>
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb]: this line should be moved to precede `Imports System.Text` -- we want the list of imports to be included in the example.

File: snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+			'<snippet3>
+			' Request authentication.
+			Dim clientStream = client.GetStream()
+			Dim authStream As New NegotiateStream(clientStream, False)
+			'</snippet1>     
+
+			' Pass the NegotiateStream as the AsyncState object 
+			' so that it is available to the callback delegate.
+			Dim ar = authStream.BeginAuthenticateAsClient(
+                New AsyncCallback(AddressOf EndAuthenticateCallback), authStream)
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb]: bad alignment

File: snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+			ar.AsyncWaitHandle.WaitOne()
+
+			' Display the properties of the authenticated stream.
+			AuthenticatedStreamReporter.DisplayProperties(authStream)
+
+			' Send a message to the server.
+			' Encode the test data into a byte array.
+			Dim message = Encoding.UTF8.GetBytes("Hello from the client.")
+			ar = authStream.BeginWrite(message, 0, message.Length, 
+               New AsyncCallback(AddressOf EndWriteCallback), authStream)
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb]: bad alignment

**Comment by obelink** [comment]
Will take care of these changes. And yes, I will do a PR to add these samples to the documentation. Already started that, but saw thate there are much more VB samples involved for related/underlying pages. Wjat do you prefer.. Page by page or Topic by Topic?

**Comment by rpetrusha** [comment]
Topic by topic probably works best, @obelink.

**Comment by rpetrusha** [comment]
I've addressed my earlier comments, @obelink, so I'll merge your PR, as well as open a PR in dotnet/dotnet-api-docs to add the example to the documentation. We'd welcome additional VB examples as you have time.

**Pull request merged**